deploy/terraform/network.tf

```diff
@@ -75,46 +75,46 @@ resource "aws_security_group" "nia_gp2gp_dmz" {
 
   ingress {
     from_port   = var.http_server_port
-    protocol    = "tcp"
+    protocol    = var.vpc_protocol_default
     to_port     = var.http_server_port
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks = [var.allow_all_ipv6_cidr_block]
   }
 
   //TODO: restrict access to match current restrictions
 
-  //ACCEPT TRAFFIC TO FACADE PORT 8081
+  //ACCEPT TRAFFIC TO FACADE PORT
   ingress {
-    from_port = 8081
-    protocol  = "TCP"
-    to_port   = 8081
-    cidr_blocks = ["0.0.0.0/0"]
+    from_port = var.FACADE_SERVER_PORT
+    protocol  = var.vpc_protocol_default
+    to_port   = var.FACADE_SERVER_PORT
+    cidr_blocks = [var.allow_all_cidr_block]
     description = "Facade Traffic"
   }
 
-  //ACCEPT TRAFFIC TO INBOUND PORT 443
+  //ACCEPT TRAFFIC TO MHS INBOUND PORT
   ingress {
-    from_port = 443
-    protocol  = "TCP"
-    to_port   = 443
-    cidr_blocks = ["0.0.0.0/0"]
+    from_port = var.MHS_INBOUND_PORT
```

Comment:
Similarly here, 443 is an HTTPS protocol and it is not specific to MHS only, hence something like "var.https_network_protocol" would reflect its nature a bit better...

Reply:
corrected